mayland-blocks/assets/svg/post-date.svg

```diff
@@ -1,9 +1,9 @@
 <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" xmlns:xlink="http://www.w3.org/1999/xlink">
     <defs>
-        <path id="a" d="M0 0h24v24H0V0z"></path>
+        <path fill="#666666" fill-opacity="1" id="a" d="M0 0h24v24H0V0z"></path>
```

Comment:
Sadly there's no better way of doing this. Loading the svg's as background images makes it impossible for us to have dynamic fill colors. Hopefully something like an [Icon block](https://github.com/WordPress/gutenberg/issues/16484) will be a better solution in cases like this